coalib/results/Result.py

```diff
@@ -108,15 +139,19 @@ def from_values(cls,
                     debug_msg='',
                     diffs: (dict, None)=None,
                     confidence: int=100,
-                    aspect: (aspectbase, None)=None):
+                    aspect: (aspectbase, None)=None,
+                    message_arguments: dict={},
+                    ):
```

Comment:
Actually the same thing is being done on 
https://github.com/coala/coala/pull/3479/files#diff-ecc13c4181b64ed6647997adcf78200dR62
It is more of a code style thing than syntax thing.